src/lib/ngsi/ContextAttribute.cpp

```diff
@@ -770,8 +769,7 @@ std::string ContextAttribute::toJson(bool isLastElement, const std::string& rend
   //   Perhaps these two parameters (renderMode and notifyFormat) should be unified ...
   //
   if (rMode == RENDER_MODE_UNIQUE_VALUES)
-  {
-    uniqueValues = true;
+  {    
     rMode        = RENDER_MODE_VALUES;  // FIXME PR: is this correct?
   }
```

Comment:
Yes. However, I understand that as long as the change is exactly the same, there wouldn't be any merging problem.

NTC